Dockerfile

```
@@ -1,163 +1,100 @@
-    rm -rf /var/lib/apt/lists/*
-
 WORKDIR /app
 
-# Copy the full release directory from the builder stage to /app/wanderer_notifier
-COPY --from=builder /app/_build/prod/rel/wanderer_notifier /app/wanderer_notifier
-
-# Create necessary directories with appropriate permissions
-RUN mkdir -p /app/data/cache /app/data/backups /app/etc && \
-    chmod -R 777 /app/data
+# Install wget for health‐checks
+RUN apt-get update \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)